core/src/com/unciv/ui/screens/multiplayerscreens/MultiplayerScreen.kt

```diff
@@ -413,11 +415,9 @@ class MultiplayerScreen : PickerScreen() {
         }
 
         // is it our turn?
-        resignButton.isEnabled = multiplayerGame.preview?.getCurrentPlayerCiv()?.playerId == game.settings.multiplayer.getUserId()
-
-        val preview = multiplayerGame.preview
-        // the latter checks if we are on the first turn on a new game, where the start time has not yet been set (default 0L)
-        if (resignButton.isEnabled || preview == null || preview.currentTurnStartTime == 0L){
+        resignButton.isEnabled = preview?.getCurrentPlayerCiv()?.playerId == game.settings.multiplayer.getUserId()
+        
+        if (resignButton.isEnabled || preview == null){
```

Comment:
```suggestion
        if (resignButton.isEnabled || preview == null) {
```
Yea that was missing before but still...